models/zone/src/zone-draft/presets/sample-data-fashion/country-spain-and-germany.ts

```diff
@@ -1,7 +1,9 @@
-import ZoneDraft from '../../builder';
+import { TZoneDraftBuilder } from '../../../types';
+import * as ZoneDraft from '../../index';
 
-const withCountrySpainAndGermany = () =>
-  ZoneDraft()
+const withCountrySpainAndGermany = (): TZoneDraftBuilder =>
```

Comment:
nit: this is the only preset prepended with `with*`

Reply:
good catch, fixed